cmd/process-agent/app/events.go

```diff
@@ -64,13 +81,26 @@ func runEventListener(cmd *cobra.Command, args []string) error {
 		return log.Criticalf("Error parsing config: %s", err)
 	}
 
+	return nil
+}
+
+func printEvent(e *model.ProcessEvent) {
+	b, err := json.MarshalIndent(e, "", "  ")
```

Comment:
nit: Eventually we should probably format using a go template like we do for process/container checks?

Reply:
I'm completely in favor of that. I haven't done it in this PR to keep it focused on the Store logic. I'll revisit the output when we have the `process_events` in the check command. Wdyt?